pkg/cmd/adm/unregister_member_test.go

```diff
@@ -164,9 +164,11 @@ func TestUnregisterMemberLacksPermissions(t *testing.T) {
 	AssertToolchainClusterSpec(t, fakeClient, toolchainCluster)
 }
 
-func mockRestart(ctx *clicontext.CommandContext, clusterName string) error {
-	if clusterName == "host" && ctx != nil {
+func mockRestart(ctx *clicontext.CommandContext, clusterName string, cfc ConfigFlagsAndClientGetterFunc) error {
+	_, _, cfcerr := cfc(ctx, clusterName)
+	if clusterName == "host" && ctx != nil && cfcerr == nil {
```

Comment:
do we event need to check for `clusterName == "host" ` , it's only being called with `member1` as an argument If I'm seeing it correctly.

Reply:
its just checking that when restart is being called, cluster name which is being passed is host